tests/installation/partitioning_filesystem.pm

```diff
@@ -20,9 +20,11 @@ use partition_setup 'unselect_xen_pv_cdrom';
 sub run {
 
     my $fs = get_var('FILESYSTEM');
+    $cmd{guidedsetup} = 'alt-g' if (is_storage_ng());
```

Comment:
We don't need this, as we have it [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning.pm#L32).